### DISCUSSION TITLE
[Long-term idea] Mentorship system

### DISCUSSION THREAD

**Discussion by ghost** [fathomnet/community-feedback#15] [Ideas]
This is a very long-term idea (and one largely dependent on time, community buy-in, and probably funding) but I've been encouraged to share it. After reflecting on last week's discussion about levels of expertise and verification, coupled with some of the concerns by taxonomists, I think it would be great in the future to have a FathomNet-sponsored mentorship program of sorts in which more junior annotators are paired with more senior experts. For example, if an aspiring biologist is heavily interested in ctenophores, (and if he agreed) they could have Dr. George Matsumoto as a mentor or be part of a smaller working group dedicated to ctenophores. This could potentially make the verification process more efficient, give experts an opportunity to inspire future scientists, and equip junior contributors with the knowledge needed to make accurate annotations.

Also in line with @kakanikatija's vision for community tutorials, @jeffday also suggested that experts could be encouraged to create their own identification guides. I know that there are a bunch out there already, but perhaps these can be tailored specifically for users who might be newer to biology and taxonomy.

**Comment by kakanikatija** [comment]
I think this is a great idea @tropicofdan. We could almost envision a multi-tiered approach: new enthusiasts could be grouped together, generating annotations on the same video clips at higher taxonomic levels (class); viewing all the user layers, a more experienced enthusiast can review progress, provide input, verify the labels, and add more fine-grained classifications if known; once this verification process is done, data can be uploaded to FathomNet. For more experienced enthusiasts and/or junior taxonomists, we can definitely link them up with taxonomy mentors while newer fine-grained classifications are added. Once these individuals get signed off on their mentorship that identifies their area of expertise, verification access can be granted to FathomNet for that emphasis in the taxonomic hierarchy.
